app.js

```diff
@@ -1,7 +1,14 @@
 
 const type=["archer","axeman","swordman"];
```

Comment:
looks good, consider using a global object to store all the relevant images, types etc